src/auth/auth.ts

```diff
@@ -104,24 +104,28 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> implements BaseAuthI
   }
 
   /**
-   * Verifies a JWT auth token. Returns a Promise with the tokens claims. Rejects
-   * the promise if the token could not be verified. If checkRevoked is set to true,
-   * verifies if the session corresponding to the ID token was revoked. If the corresponding
-   * user's session was invalidated, an auth/id-token-revoked error is thrown. If not specified
-   * the check is not applied.
+   * Verifies a JWT auth token. Returns a Promise with the tokens claims.
```

Comment:
This should be lowercase unless it's a (backticked) literal, I think.

Reply:
Hey @egilmorez , I think there are 16 JWT being used in this file and 50 times+ in other files as well, along I agree with you, but since that wasn't my change, I am not I should change those.